doc/reference.md

```diff
@@ -262,6 +265,57 @@ try {
 ```
 
 
+---
+
+### startStoreCard
+
+Used to start the store a card for a customer flow.
+
+The card information is stored on Square servers, not on the specific device running Reader SDK. This means cards cannot be saved on file when offline, and that saved cards for a customer are available from any device, keyed by customer ID.
```

Comment:
Style issue: avoid "This" if possible. Suggest: 

Used to start the store a card for future purchases.

Card information is **only** stored on Square servers, not on the mobile device running Reader SDK, which means cards cannot be saved while the device is offline. Saved card information can be requested using Square APIs and the associated customer ID.